node/device/node-red/azureiothub/azureiothub.js

```diff
@@ -0,0 +1,162 @@
+module.exports = function (RED) {
+
+    var Client = require('azure-iot-device').Client;
+
+    var Protocols = {
+        amqp: require('azure-iot-device-amqp').Amqp,
+        mqtt: require('azure-iot-device-mqtt').Mqtt,
+        http: require('azure-iot-device-http').Http,
+        amqpWs: require('azure-iot-device-amqp-ws').AmqpWs
+    };
+
+    var Message = require('azure-iot-device').Message;
+
+    var client = null;
+    var clientConnectionString = "";
+    var clientProtocol = "";
+    var pendingMessage = null;
+    var node = null;
+    var nodeConfig = null;
+
+    var statusEnum = {
+        disconnected: { color: "red", text: "Disconnected" },
+        connected: { color: "green", text: "Connected" },
+        sent: { color: "blue", text: "Sent message" },
```

Comment:
I have not found any other place to put these. I'll keep them there for now, but I agree that Connected and Disconnected should be the only statuses for the node